pay-api/migrations/versions/2024_03_15_fb3ba97b603a_new_version_tables_.py

```diff
@@ -0,0 +1,202 @@
+"""Migration for new history tables
+
+Revision ID: fb3ba97b603a
+Revises: bacb2b859d78
+Create Date: 2024-03-15 15:22:53.140353
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'fb3ba97b603a'
+down_revision = 'bacb2b859d78'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
There's still 2 lines that show up on migrate, not sure why.. index on invoice_reference and disbursement field on partial refunds